README.md

```diff
@@ -1,77 +1,71 @@
-This instruction is going to have opcode 0b111111, plus three bits for the first register, and three more for the second, resulting in 0b111111xxxyyydddd (where x is the first bits for the first register, y is for the second, and d is a don't care value, meaning an unused 0 or 1).
-
-The entry in AllInstructions will be `{OpINCMOD, genRegM("incmod", 2), 1, execINCMOD},`, where OpINCMOD is defined above in the constant block as `OpINCMOD = 0b111111`. `genRegM` is used with those arguments to define the mnemonic and to say that it has two register opcodes in the usual position. 1 is to say that a single word is necessary to encode the instruction. execINCMOD is defined as follows:
-
-```golang
-func execINCMOD(pr *ICMCProcessor) error {
-    // get the instruction binary data
-    inst := pr.Data[pr.PC]
-
-    // remember, those are indices from 0-7, not the actual values
-    rx_index := getRegAt(inst, 7) // 7 is the first bit from right to left encoding the register index for rx
-    ry_index := getRegAt(inst, 4) // same for ry
-
-    // set the value in rx to the result of our operation
-    pr.GPRRegs[rx_index] = (pr.GPRRegs[rx_index] + 1) % pr.GPRRegs[ry_index]
-
-    // no errors happend
-    return nil
-}
-```
-
-# What you can do to help the GO ICMC Simulator's development
-An open source project is never complete. Please help the project by submitting issues and pull requests! They will be happly accepted if they help the overall project. Some examples of what can be done:
-- Fully complete the MIF syntax parsing in the MIF package and official quartus documentation.
-- Increase processor speed by changing processor.fetchInstruction, for now the instruction fetching based on opcodes is the main bottleneck for performace.
-- Create a default "hello world" MIF code and character set and use it if the user did not pass any files themselves. To do that take a look at [the embed package](https://pkg.go.dev/embed).
-- Add a right click options menu for each instruction in the list to edit memory in place or add breakpoints. This is possible using a new type and go struct inheritance in the instructionList and some remodeling in processor.RunUntilHalt.
-- Documentation of instruction execution and mnemonic generation.
-
-# Contributors
-The main contributors to the project are listes here:
-- Lucas Eduardo Gulka Pulcinelli ([github](https://github.com/lucasgpulcinelli))
-
-Special thanks to:
-- Artur Brenner Weber ([github](https://github.com/ArturWeber)), for providing macOS/arm64 builds and helping with documentation
-- Daniel Contente Romanzini ([github](https://github.com/Dauboau)), for providing macOS/amd64 builds
+# GO ICMC Simulator 🖥️
+
+![GO ICMC Simulator](https://github.com/lucasgpulcinelli/goICMCsim/assets/11618151/da81d732-5cb4-4f41-9128-37ae864ceac9)
```

Comment:
Pera, você poderia me explicar isso? Eu entendi que esse é o link para a imagem, mas como isso foi gerado, e onde esse asset foi definido? Tem documentação sobre para eu dar uma olhada? Gostei bastante

Reply:
Acho q é uma "bugfeature" do GitHub KKKKKK
Aqui nas issues vc consegue fazer upload de imagem e pra isso ele faz upload dessa imagem. Só que como usa markdown aqui tbm ele coloca o link da imagem que foi feito upload e aí eu copiei e usei esse link.

Reply:
![goICMCsim_08_T](https://github.com/lucasgpulcinelli/goICMCsim/assets/11618151/06b9107e-835d-450e-84ec-94e74da7e19c)

Reply:
Outra versão da capa.
Mas devem existir outras formas de fazer isso tbm. Porque no repositório tem como definir uma imagem de [Social Preview](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview). Inclusive essas imagens já estão na resolução correta pra ser usadas tbm se vc quiser.